folium/features.py

```diff
@@ -652,6 +682,69 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
         self.className = class_name
 
 
+class Tooltip:
+    """
+    Creates a Tooltip object for adding to features to display text as a property a Map by executing a javascript
+    function when hovering the cursor over each feature.
+
+    Parameters
+    ----------
+        fields: list or tuple.
+            Labels of the GeoJson 'properties' or GeoPandas GeodataFrame columns you'd like to display.
+        aliases: list or tuple
+            Optional 'aliases' you'd like to display the each field name as, to describe the data in the tooltip.
+            *Must consist of strings, and be of the same length as fields.
+        text: str
+            Pass the same string as a tooltip for every value in the GeoJson object, I.e. "Click for more info."
+            *May not be passed if fields is not None
+        labels: boolean True or False
```

Comment:
You don't have to specify that a boolean is True or False. I would suggest to write:

`labels : bool, default True`